### DISCUSSION TITLE
[Weekly Review Report]: Jan 27 - Feb 2

### DISCUSSION THREAD

**Discussion by ashviyer** [wikipathways/wikipathways-help#102] [curation-reports]
### Recent changes

WP3859 - Added protein complex object and added pathway identifier for SMAD pathway
WP1812- eif1 identifier added.
WP4629 - Added  a commnet for the Lactase transporter - Monocarboxylate transporters , the specific types being tissue-specific. Oxygen transport gene in the pathway - isnt it based on diffusion rather than  a transporter? 
WP4725 - added identifiers, graphical elements added for multi datanodes
WP5440- protein complex graphical elements out of bounds with the datanodes
WP5441- connection between tyrosine kinase and PI3K- phosphorylation?
All other were aapproved pathway with appropriate tags.

### Under construction

No pathways for this category.

### New pathways

No pathways for this category.

### Deleted Pathways

No pathways for this category.

### Additional notes

Opened an issue for WP4629  - to clarify the issue with oxygen transporters in the pathway 
Opened an issue for WP5440- link between receptor tyrosine kinase and PI3K

### Update Approved pathway tag

- [ ] All Approved pathways tags were updated!

**Comment by Chris-Evelo** [comment]
There is a comment in WP4629  on the left side about applies to PKA etc, it is not easy to understand that

**Reply by ashviyer** [reply]
Ah, yes! I did see it now. Alteast from the interactions I see that PKA and PKL can convert Fructose bi-phosphate to pyruvate but not PKM. But even then the oxygen transport would be based on diffusion given how the interaction are specified here.

**Reply by Chris-Evelo** [reply]
My question was more basic than that. For me it is not really clear from that comment what PKA, PKL and PKM are. I am not an expert on oxygen transport into the cell,. but I understand that there is also facilitated diffusion, which typically involves a transporter that doesn't consume energy. Maybe that is what this is about?
